<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Issu</class>
 <widget class="QMainWindow" name="Main_Issu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>991</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>시도교육청</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_location">
     <property name="geometry">
      <rect>
       <x>94</x>
       <y>15</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="currentText">
      <string>서울특별시교육청</string>
     </property>
     <item>
      <property name="text">
       <string>서울특별시교육청</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>부산광역시교육청</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>대구광역시교육청</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>인천광역시교육청</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>광주광역시교육청</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>대전광역시교육청</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>울산광역시교육청</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>세종특별자치시교육청</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>경기도교육청</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>강원도교육청</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>충청북도교육청</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>충청남도교육청</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>전라북도교육청</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>전라남도교육청</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>경상북도교육청</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>경상남도교육청</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>제주도교육청</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>학교명</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_search">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>10</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>검색</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_scname">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>15</y>
       <width>570</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>991</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="table_result">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>961</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>3</number>
     </property>
     <property name="columnCount">
      <number>6</number>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>120</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>번호</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>학교명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>관할 지역청</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>공/사립</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>우편번호</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>도로명 주소</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="0" column="4">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="0" column="5">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="1" column="3">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="1" column="4">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="1" column="5">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="2" column="2">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="2" column="3">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="2" column="4">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="2" column="5">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="info_result">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>14</y>
       <width>951</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>검색 결과</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>991</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="table_output">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>961</width>
       <height>241</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>6</number>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>120</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>번호</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>학교명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>관할 지역청</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>공/사립</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>우편번호</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>도로명 주소</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QPushButton" name="button_del">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>290</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>∧</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_insert">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>290</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>∨</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_output">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>610</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>내보내기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_reset">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>610</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>초기화</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1012</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_search</sender>
   <signal>clicked()</signal>
   <receiver>Main_Issu</receiver>
   <slot>search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>949</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>1011</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_del</sender>
   <signal>clicked()</signal>
   <receiver>Main_Issu</receiver>
   <slot>delete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_insert</sender>
   <signal>clicked()</signal>
   <receiver>Main_Issu</receiver>
   <slot>insert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>555</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_output</sender>
   <signal>clicked()</signal>
   <receiver>Main_Issu</receiver>
   <slot>output()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>946</x>
     <y>652</y>
    </hint>
    <hint type="destinationlabel">
     <x>665</x>
     <y>645</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_reset</sender>
   <signal>clicked()</signal>
   <receiver>Main_Issu</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>862</x>
     <y>656</y>
    </hint>
    <hint type="destinationlabel">
     <x>512</x>
     <y>650</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>search()</slot>
  <slot>delete()</slot>
  <slot>insert()</slot>
  <slot>output()</slot>
  <slot>reset()</slot>
 </slots>
</ui>
